locations/spiders/zoloto_585_ru.py

```
@@ -1,27 +1,40 @@
 
+from locations.categories import Categories, apply_category
+from locations.hours import DAYS_RU, NAMED_DAY_RANGES_RU, OpeningHours
 from locations.items import Feature
-from locations.user_agents import BROWSER_DEFAULT
+from locations.json_blob_spider import JSONBlobSpider
 
 
-class Zoloto585RUSpider(SitemapSpider):
+class Zoloto585RUSpider(JSONBlobSpider):
     name = "zoloto_585_ru"
     item_attributes = {"brand_wikidata": "Q125730875"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)